config.yaml

```diff
@@ -1147,6 +1147,10 @@ Images:
   - 15.6.19.1
   - 15.6.24.2
   TargetImageName: mirrored-bci-micro
+- SourceImage: registry.suse.com/rancher/elemental-operator
+  Tags:
+  - 1.6.9
+  TargetImageName: elemental-operator
```

Comment:
if we're adding something new, shouldn't it have the mirrored- prefix?

Reply:
@fgiudici and I talked about this over slack. TLDR: he wants to mirror `registry.suse.com/rancher/elemental-operator` to `docker.io/rancher/elemental-operator` for simplicity's sake. Otherwise we are mirroring it to `docker.io/rancher/mirrored-elemental-operator` and `registry.suse.com/rancher/mirrored-elemental-operator`, which can be confusing because we are mirroring the image back to the registry it came from. Whether the `mirrored-` prefix applies is unclear, since we produce `elemental-operator` internally. AFAIK the current setup is fine, but I might not know something @fgiudici does.

Let me know if you want more of the history of the elemental images + image-mirror.